census-us/Readme.md

```diff
@@ -19,10 +19,13 @@ For keeping the sources and reuse it (beware of any US Census update...):
 2> At the next build (of the same or another task) , check: "use previous sources".
 
 ## 1/ US CENSUS CUSTOM DATASET:
-Enter your states in lower case separated by a comma with no space.
+Enter your states in lower case separated by a comma with no space. Need help on that? Please have a look at the custom dataset: US Census states resources dataset
 Define the level required (TRACT, COUNTY, BLOCK_GROUP)
-Enter manually the Census fields you would get (ex: B00001_001E,B08534_001E). Here is the total catalog : http://api.census.gov/data/2014/acs5/variables.html 
+Enter manually the Census fields you would get (ex: B00001_001E,B08534_001E). Here is the total catalog : https://api.census.gov/data/2014/acs/acs5/variables.html 
 - For a more agnostic selection, please us the "US Census feature selection recipe".
+Click test.
+- It is expected that the dataset may take a while to build, this is because the US Census files are built in background. Don't close the test window. When done, click create dataset. It's highly recommended to sync this dataset into another one, data won't change regularly.
+- As usual, check the number of rows returned in the final dataset broken down by state before using the dataset created in your work.
```

Comment:
could we add validations for some representative sample of states so users don't have to assert? Not sure if this works with the flow of the plugin but might be nice especially if this is a common pitfall